tests/ui/test_search.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from pages.desktop.home import Home
+
+
+@pytest.mark.nondestructive
+def test_that_page_with_search_results_has_correct_title(
+        my_base_url, selenium, initial_data):
+    """Test the search term entered loads on the search page."""
+    home_page = Home(selenium, my_base_url).open()
+    search_keyword = 'Search term'
+    home_page.search_for(search_keyword)
+    assert search_keyword in selenium.title
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_that_searching_for_addon_returns_addon_as_first_result(
+        my_base_url, selenium, addon):
+    """Test searching for an addon returns the addon."""
+    home_page = Home(selenium, my_base_url).open()
+    name = home_page.featured_extensions.extensions[0].name
+    search_page = home_page.search_for(name)
```

Comment:
In that case could we do something like the following to use the addon name if we have one:

```python
name = getattr(addon, 'name', home_page.featured_extensions.extensions[0].name)
```